projects/cert-manager/cert-manager/README.md

```diff
@@ -0,0 +1,63 @@
+## **cert-manager**
+![Version](https://img.shields.io/badge/version-v1.5.3-blue)
```

Comment:
version is not updated here

Reply:
Done, probably that validator failed that I added.